sdk/core/System.ClientModel/api/System.ClientModel.net6.0.cs

```diff
@@ -70,6 +70,20 @@ protected ContinuationToken(System.BinaryData bytes) { }
         public static System.ClientModel.ContinuationToken FromBytes(System.BinaryData bytes) { throw null; }
         public virtual System.BinaryData ToBytes() { throw null; }
     }
+    public partial class Credential
```

Comment:
this name is too generic. Also, as I noted above, I don't like the drastic terminology change from Azure.Core. I think we somehow have to call this type <something>Token and the provider <something>Credential.

Reply:
I agree that it is a big terminology change, but it seems more correct to what they do.